devel/meson/Portfile

```diff
@@ -4,30 +4,29 @@ PortSystem          1.0
 PortGroup           python 1.0
 PortGroup           github 1.0
 
-github.setup        mesonbuild meson 0.59.3
+github.setup        mesonbuild meson 0.62.1
 revision            0
 
 github.tarball_from releases
 license             Apache-2
 categories          devel python
 maintainers         {soap.za.net:git @SoapZA} openmaintainer
-platforms           darwin
 supported_archs     noarch
 installs_libs       no
 homepage            https://mesonbuild.com
 
-description         Meson - A high productivity build system
+description         Meson – a high productivity build system
 long_description    Meson is a build system designed to optimize programmer productivity. \
                     It aims to do this by providing simple, out-of-the-box support for modern \
                     software development tools and practices, such as unit tests, coverage reports, \
                     Valgrind, CCache and the like. It is both extremely fast, and, even more importantly, \
                     as user friendly as possible.
 
-checksums           rmd160  36994a8e29e15751a9a1977aad581057a52f3943 \
-                    sha256  0a35cfd4cb779a6a415ad3ccbf2d4143334e96139125585ce49915bec494da02 \
-                    size    1953620
+checksums           rmd160  96db39f90328a64d508cc7c0075aa89fecde939b \
+                    sha256  a0f5caa1e70da12d5e63aa6a9504273759b891af36c8d87de381a4ed1380e845 \
+                    size    2034805
 
-# as of verison 0.45.0,requires python 3.5 or better
+# as of verison 0.45.0, requires python 3.5 or better
```

Comment:
Probably better to just update this note
```
as of verison 0.57.0, requires python 3.6 or later
```

Reply:
@Gcenx Good point, thanks! Will do it.

Reply:
I’d screwed up and not pushed my comments, this should actually be updated to 0.62.0 and required python 3.7 and later

Then remove python36 from the list

Reply:
> I’d screwed up and not pushed my comments, this should actually be updated to 0.62.0 and required python 3.7 and later
> 
> Then remove python36 from the list

Done already.

Reply:
> > I’d screwed up and not pushed my comments, this should actually be updated to 0.62.0 and required python 3.7 and later
> > Then remove python36 from the list
> 
> Done already.

I’d seen don’t worry it’s GitHub review was acting up, that was just incase it didn’t post then correctly.